### DISCUSSION TITLE
algebra relazionale - 4

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#61] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/a1e069ee-0aeb-49a6-8634-12f6196518c5)

**Comment by Near047** [comment]
![photo_2025-02-09_12-34-19](https://github.com/user-attachments/assets/c33614fa-b39d-4343-8786-17a46a2d6810)

Non so se sia corretta ma questa è la mia risposta
